js_modules/dagit/packages/core/src/partitions/BackfillMessaging.tsx

```diff
@@ -0,0 +1,152 @@
+import {Alert, ButtonLink, Colors, Group, Mono} from '@dagster-io/ui';
+import {History} from 'history';
+import * as React from 'react';
+
+import {showCustomAlert} from '../app/CustomAlertProvider';
+import {SharedToaster} from '../app/DomUtils';
+import {PythonErrorInfo} from '../app/PythonErrorInfo';
+import {graphql} from '../graphql';
+import {
+  DaemonNotRunningAlertInstanceFragmentFragment,
+  LaunchPartitionBackfillMutation,
+  UsingDefaultLauncherAlertInstanceFragmentFragment,
+} from '../graphql/graphql';
+
+const DEFAULT_RUN_LAUNCHER_NAME = 'DefaultRunLauncher';
+
+function messageForLaunchBackfillError(data: LaunchPartitionBackfillMutation | null | undefined) {
+  const result = data?.launchPartitionBackfill;
+
+  let errors = <></>;
+  if (result?.__typename === 'PythonError' || result?.__typename === 'PartitionSetNotFoundError') {
+    errors = <PythonErrorInfo error={result} />;
+  } else if (result?.__typename === 'InvalidStepError') {
+    errors = <div>{`Invalid step: ${result.invalidStepKey}`}</div>;
+  } else if (result?.__typename === 'InvalidOutputError') {
+    errors = <div>{`Invalid output: ${result.invalidOutputName} for ${result.stepKey}`}</div>;
+  } else if (result && 'errors' in result) {
+    errors = (
+      <>
+        {result['errors'].map((error, idx) => (
+          <PythonErrorInfo error={error} key={idx} />
+        ))}
+      </>
+    );
+  }
+
+  return (
+    <Group direction="column" spacing={4}>
+      <div>An unexpected error occurred. This backfill was not launched.</div>
+      {errors ? (
```

Comment:
Looks like `errors` will always be truthy, since the default is a fragment.

Reply:
Wow good catch, will change this! I wonder how long it's been like this, was just moving it in this PR